src/shared/colors.ts

```diff
@@ -21,6 +21,8 @@ enum Colors {
   YELLOW_DARK = '#FFD167',
   YELLOW_LIGHT = '#FFECB8',
   WHITE = '#FFFFFF',
+  DARK_BACKGROUND = '#181818',
```

Comment:
it's already exist as `GREY_DARK`

Reply:
I didn't use it and create it.  Button component uses it.

Reply:
one minor missed - please do not create the same variable here and use `GREY_DARK `